src/Scaler.Demo/Shared/CosmosDbConfig.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Keda.CosmosDb.Scaler.Demo.Shared
+{
+    public class CosmosDbConfig
+    {
+        public string Connection { get; set; }
+        public string DatabaseId { get; set; }
+        public string ContainerId { get; set; }
+        public int ContainerThroughput { get; set; }
+
+        public string LeaseConnection { get; set; }
+        public string LeaseDatabaseId { get; set; }
+        public string LeaseContainerId { get; set; }
```

Comment:
Wouldn't it be better if we introduce a `Lease` object that has these fields?
```suggestion
        public string LeaseConnection { get; set; }
        public string LeaseDatabaseId { get; set; }
        public string LeaseContainerId { get; set; }
```